src/carbonio-files-ui-common/utils/utils.ts

```diff
@@ -144,6 +144,14 @@ export const getIconColorByFileType = (
 			return theme.palette.success.regular;
 		case NodeType.Application:
 			return theme.palette.gray0.regular;
+		case NodeType.Root: {
+			switch (subType) {
+				case ROOTS.SHARED_WITH_ME:
+					return '#AB47BC';
```

Comment:
If the icon should change, and the color too, do not use the hardcoded string. Replace all the usages of the hardcoded string with the theme token provided by the shell (theme.palette.linked.regular, which needs to be added also inside the styled-components.d.ts)